cmd/api/databasehandler.go

```diff
@@ -434,29 +437,42 @@ func CountOp(task *tasks.GenericLoadingTask) (string, int64, bool) {
 			resultString = "Table name not provided"
 			break
 		}
-		cassClusterConfig := gocql.NewCluster(task.ConnStr)
-		cassClusterConfig.Authenticator = gocql.PasswordAuthenticator{Username: task.Username, Password: task.Password}
-		cassClusterConfig.Keyspace = task.Extra.Keyspace
-		cassandraSession, errCreateSession := cassClusterConfig.CreateSession()
-		if errCreateSession != nil {
-			log.Println("Unable to connect to Cassandra! err:", errCreateSession.Error())
-			resultString += "Unable to connect to Cassandra! err:" + errCreateSession.Error()
-			break
-		}
-		defer cassandraSession.Close()
-
-		countQuery := "SELECT COUNT(*) FROM " + task.Extra.Table
-		var rowCount int64
-		if errCount := cassandraSession.Query(countQuery).Scan(&rowCount); errCount != nil {
-			log.Println("Error while getting COUNT", errCount)
+		//cassClusterConfig := gocql.NewCluster(task.ConnStr)
```

Comment:
If you are not going to use this code, then delete it, instead of commenting it out,